experimental-examples/tina-cloud-starter/.tina/__generated__/_graphql.json

```diff
@@ -642,20 +642,6 @@
                   "value": "Int"
                 }
               }
-            },
-            {
-              "kind": "InputValueDefinition",
-              "name": {
-                "kind": "Name",
-                "value": "filter"
-              },
-              "type": {
-                "kind": "NamedType",
-                "name": {
-                  "kind": "Name",
-                  "value": "DocumentFilter"
-                }
-              }
```

Comment:
I noticed some of the `filter` stuff got added to the experimental `tina-cloud-starter`?  I'm guessing that's from the DataStore PR.  I don't typically test `TinaAdmin` with that, so this PR generated without it.

Reply:
Yeah, this is fine to remove, it's just whether you ran the CLI with `--experimentalData` or not unfortunately. I'll try to remember to leave that out until we're ready for it